<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cuda_Oglnew_Form</class>
 <widget class="QWidget" name="Cuda_Oglnew_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="cudanew_class" name="ogl_widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>731</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="xSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="ySlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="speedSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="amplitSlider">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>16</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="colorSlider">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>16</width>
     <height>451</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cudanew_class</class>
   <extends>QWidget</extends>
   <header location="global">cudanew_class.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
